sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -546,8 +540,17 @@ object Radar {
         val usage = "initialize"
         this.apiClient.getConfig(usage, false, object : RadarApiClient.RadarGetConfigApiCallback {
             override fun onComplete(status: RadarStatus, config: RadarConfig) {
-                locationManager.updateTrackingFromMeta(config?.meta)
-                RadarSettings.setFeatureSettings(context, config?.meta.featureSettings)
+                locationManager.updateTrackingFromMeta(config.meta)
+                RadarSettings.setFeatureSettings(context, config.meta.featureSettings)
+                RadarSettings.setSdkConfiguration(context, config.meta.sdkConfiguration)
+
+                val sdkConfiguration = RadarSettings.getSdkConfiguration(context)
```

Comment:
Do we really need to set something and then get the same thing in a function? Won't it be more efficient to just use the object returned from the API?

Reply:
config.meta.sdkConfiguration might be null, then the set would be no-op, then we get the existing real sdkConfig